src/mudlet.cpp

```diff
@@ -2067,9 +2067,12 @@ void mudlet::slot_show_help_dialog_download()
 
 void mudlet::slot_show_about_dialog()
 {
-    auto pDlg = new dlgAboutDialog( this );
-    pDlg->raise();
-    pDlg->show();
+    if( ! mpAboutDlg ) {
+        mpAboutDlg = new dlgAboutDialog( this );
```

Comment:
Is there a reason to force this and not let Qt manage the resources?

It would be more efficient to re-use the cached dialog instead of forcing a deletion every time.